### ISSUE TITLE
Replace OAuth2ModuleFactory break by default one

### ISSUE THREAD

**Issue by DamnClin** [jhipster/jhipster-lite#2347]

**Comment by pascalgrimaud** [comment]
Don't understand this ticket, can you explain it plz?

**Comment by DamnClin** [comment]
Yep, the BREAK constant in https://github.com/jhipster/jhipster-lite/blob/main/src/main/java/tech/jhipster/lite/generator/server/springboot/mvc/security/oauth2/domain/OAuth2ModuleFactory.java needs to be replaced by the one from JHipsterModule
